modules/ossm-servicemesh-overview.adoc

```diff
@@ -1,21 +1,16 @@
 ////
 Module included in the following assemblies:
-* service_mesh/v1x/ossm-architecture.adoc
-* service_mesh/v2x/ossm-architecture.adoc
-* service_mesh/v1x/servicemesh-release-notes.adoc
-* service_mesh/v2x/servicemesh-release-notes.adoc
+* service_mesh/v2x/ossm-about.adoc
 ////
 
 [id="ossm-servicemesh-overview_{context}"]
-= {ProductName} overview
+= Introduction to {ProductName}
 
-{ProductName} is a platform that provides behavioral insight and operational control over the service mesh, providing a uniform way to connect, secure, and monitor microservice applications.
+{ProductName} is designed to address microservice architecture problems. A service mesh creates a centralized point to control features for many or all microservices in an application. {ProductShortName} adds a transparent layer on existing distributed applications without requiring any changes to the application code.
 
```

Comment:
Consider revision: Red Hat OpenShift Service Mesh is designed to address a variety of problems in your microservice architecture by creating a centralized point of control in your application. It adds a transparent layer on existing distributed applications without requiring any changes to the application code.

Reply:
revised